materialize-sql/validate.go

```diff
@@ -111,9 +116,7 @@ func ValidateMatchesExisting(existing *pf.MaterializationSpec_Binding, proposed
 	// mention are implicitly forbidden.
```

Comment:
The comment just above here `...and forbid any that aren't already in our map` could perhaps be updated, since that won't be the case anymore.